scripts/contributing.js

```diff
@@ -0,0 +1,36 @@
+/* global jQuery*/
+
+(function($){
+    var githubIssuesEndpoint = 'https://api.github.com/repos/babel/babel/issues';
```

Comment:
The docs mention authentication for that endpoint - and I'm getting a 404 when trying to access it (despite *expecting* a 401):

https://api.github.com/orgs/babel/issues